examples/dask_ml_simple.py

```diff
@@ -0,0 +1,76 @@
+"""
+Optuna example that optimizes a classifier configuration for the Iris dataset using Dask-ML.
+
+In this example, we optimize a logistic regression classifier configuration for the Iris dataset.
+The classifier is from dask-ml while the dataset is from sklearn.
+We optimize the choice of solver (admm, gradient descent, or newton),
```

Comment:
The objective function in this example seems using `proximal_grad` instead of `newton` as a choice.
```suggestion
We optimize the choice of solver (admm, gradient descent, or proximal gradient),
```

Reply:
Great catch. I'll fix that